### PR TITLE
Bug 1701162: Fix CSV link for installed operator

### PR DESCRIPTION
**Pull request by spadgett**
Fixes https://bugzilla.redhat.com/show_bug.cgi?id=1701162

/assign @jeff-phillips-18

### CODE REVIEW

File: frontend/public/components/operator-hub/operator-hub-item-details.tsx [hunk 1]
```diff
@@ -37,7 +37,7 @@ export const OperatorHubItemDetails: React.SFC<OperatorHubItemDetailsProps> = ({
 
   const getHintBlock = () => {
     if (installed) {
-      const filterName = _.get(item.obj, 'status.packageName', item.obj.metadata.name);
+      const filterName = _.get(item.obj, 'status.channels[0].currentCSV', '');
```
Comment (jeff-phillips-18) [inline frontend/public/components/operator-hub/operator-hub-item-details.tsx]: Looks like @alecmerdler  changed it from `channels[0].currentCSV` to `packageName` in #1327

Comment (spadgett) [reply frontend/public/components/operator-hub/operator-hub-item-details.tsx]: We could simply take the name filter out if it's not reliable.

**Comment by spadgett** [comment]
@jeff-phillips-18 I removed the name filter since it doesn't seem to work in all cases. (OLM tests were failing.) I'd rather show too many items than hide the one you're looking for.

**Pull request merged**